CHANGELOG.md

```diff
@@ -7,11 +7,22 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [v0.1.6]
 
+### Added
+
+#### xrpl
+
+- Configurable time out for the RPC client. New default time of 5 seconds instead of 1 second.
+
+#### examples
```

Comment:
We should only add package changes to `binary-codec`, `address-codec`, `keypairs` and `xrpl`